tests/kerneltest.sh

```diff
@@ -87,6 +92,7 @@ for TEST in $TESTS; do
       -o format:gob \
       -o option:parse-arguments \
       -o option:detect-syscall \
+      -o err-file:${ERROR_FILE} \
```

Comment:
We should place the stderr file in /tmp together with "/tmp/build-$$" file. OR make sure all temp files are in the same place (and with same naming convention, like using a pid attached to temp file name).

At the end of the script we should clean both files (I forgot to remove /tmp/build-$$ as well). I would use /tmp/tracee for tracee temporary files only.

Reply:
Error file is under /tmp/tracee, and there is an `rm ${ERROR_FILE}` at the end.